### DISCUSSION TITLE
mils = mrad?

### DISCUSSION THREAD

**Discussion by dezanche** [nikolaygekht/ballistic.calculator.app#2] [General]
I haven't had a chance to test out the software in detail but I'm curious why the angular units (view menu) include both mils and milliradians. Aren't they the same thing?

**Comment by o-murphy** [comment]
> I haven't had a chance to test out the software in detail but I'm curious why the angular units (view menu) include both mils and milliradians. Aren't they the same thing?

No, 1 MIL (NATO) = ~0.98 MRAD, but it's too close and mostly we neglect it

MRAD stands for Milliradian (abbreviation MIL) and represents 1⁄1000 of a radian. 0.1 MRAD/MIL is 1 cm at 100 m and is suitable for those using the metric system. If you take it exactly though, 0.1 MIL (NATO) is actually 0.098 MRAD. However, Schmidt & Bender uses MRAD as well as MIL for exactly 1 cm at 100 m. Due to the simplicity of the metric system MRAD is usually the standard for Military and LE riflescopes. This is also why it is preferred by most tactical precision shooters.

Source: https://www.schmidtundbender.de/en/service/did-you-know/127-turrets/1521-moa-mrad-mil-cm.html

**Comment by o-murphy** [comment]
> I haven't had a chance to test out the software in detail but I'm curious why the angular units (view menu) include both mils and milliradians. Aren't they the same thing?

Also we have Thousandth that have to be same as MRad and have tp be an 1/1000 of angular minute, but cause of NATO/Sweden/USSR standards difference 1 MRAD ~= 0.95 Thousandth

It all depends on how many pieces we split an angular minute,  360 degree circle have to be divided by ~6283.185

* In the USSR, the value was adopted: 1/6000.
* In NATO: 1/6400 of a circle (also called mil, short for milliradian).
* In the army of Sweden, which was not part of NATO, the most accurate definition of 1/6300 was approved.

**Reply by dezanche** [reply]
Thanks for the clarification. So mil results from an approximation of $\pi$ to divide the circle into a round number of clicks (6400 instead of $2000\pi$). For rifles I don't see any practical advantage in doing these approximations, but I read this need came from artillery.
